featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -138,9 +138,9 @@ def __init__(self,
                  drop_exact=None,
                  where_stacking_limit=1):
 
-        if target_entity_id not in entityset.entity_dict:
+        if target_dataframe_name not in entityset.entity_dict:
```

Comment:
perhaps out of scope for this PR, but `entity_dict` has been replaced with `dataframe_dict`

Reply:
updated